src/args/argument_list.go

```diff
@@ -27,6 +27,7 @@ type ArgumentList struct {
 	CustomMetricsConfig          string `default:"" help:"YAML configuration with one or more SQL queries to collect custom metrics"`
 	ShowVersion                  bool   `default:"false" help:"Print build information and exit"`
 	ExtraConnectionURLArgs       string `default:"" help:"Appends additional parameters to connection url. Ex. 'applicationintent=readonly&foo=bar'"`
+	EnableDiskMetricsInBytes     bool   `default:"true" help:"Enable collection of disk space metrics in bytes."`
```

Comment:
The only thing I do not like is that we are adding an extra flag, next time we will need to add a further one. Changing the slice to a map we could easily build a generic way to drop things.

If Kai is interested into it we could add such feature instead